packages/base-styles/_colors.scss

```diff
@@ -25,6 +25,12 @@ $light-gray-200: #f3f4f5;
 $light-gray-100: #f8f9f9;
 $white: #fff;
 
+// G2 colors.
+$border-color: #1e1e1e;
+$theme-color: theme(button);
+$blue-medium-focus: #007cba;	// @todo: consider revisiting for theme colors.
```

Comment:
Should we fix this. I believe we can solve this with scss `tint` or `shade` like we do in the buttons style

Reply:
Because colors will be a separate topic, I feel like this one is worth shelving for now. 

One challenge with how we use the existing themes is that with the Midnight theme you get red buttons, making the red color meaningless for the trash button. So I also think there's a larger conversation here.